x/upgrade/go.mod

```diff
@@ -189,4 +213,9 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
-replace github.com/cosmos/cosmos-sdk => ../../.
+replace (
+	// celestia-core
+	github.com/cometbft/cometbft => github.com/celestiaorg/celestia-core v0.38.11-0.20250325150452-89ef6e8a5cb3 // marko/core_changes_v3
```

Comment:
random note that all these aws imports reminds me we need to merge https://github.com/celestiaorg/celestia-core/pull/1437

Reply:
I was going to ask about them on the core PR actually!